ci-operator/config/openshift/multiarch-tuning-operator/openshift-multiarch-tuning-operator-v1.0__ocp419.yaml

```diff
@@ -0,0 +1,175 @@
+base_images:
+  base:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-enterprise-base-multi-openshift-4.17
+  builder:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.22-builder-multi-openshift-4.17
+binary_build_commands: NO_DOCKER=1 make build
+build_root:
+  from_repository: true
+images:
+- additional_architectures:
+  - arm64
+  dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    builder:
+      as:
+      - golang:1.22
+  to: multiarch-tuning-operator
+- additional_architectures:
+  - arm64
+  dockerfile_path: bundle.Dockerfile
+  to: multiarch-tuning-op-bundle
+- additional_architectures:
+  - arm64
+  dockerfile_literal: |
+    FROM quay.io/operator-framework/operator-sdk:v1.31.0 as osdk
+    FROM src
+    COPY --from=osdk /usr/local/bin/operator-sdk /usr/local/bin/
+  from: src
+  to: operator-sdk-mto
+operator:
+  bundles:
+  - as: multiarch-tuning-operator-bundle
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: registry.ci.openshift.org/origin/multiarch-tuning-operator:v1.0
+    with: pipeline:multiarch-tuning-operator
+releases:
+  initial:
+    integration:
+      name: "4.19"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.19"
+      namespace: ocp
+  multi-initial:
+    candidate:
+      architecture: multi
+      product: ocp
+      stream: nightly
+      version: "4.19"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  lint:
+    limits:
+      memory: 6Gi
+    requests:
+      cpu: 500m
+      memory: 1Gi
+  sast:
+    limits:
+      memory: 6Gi
+    requests:
+      cpu: 300m
+      memory: 400Mi
+tests:
+- as: e2e-gcp
+  capabilities:
+  - arm64
+  skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Za-z]+\.md$)|((^|/)\.github/)|((^|/)\.tekton/)|((^|/).*konflux\.Dockerfile$)
+  steps:
+    cluster_profile: gcp
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-initial
+      OPERATOR_IMAGE: pipeline:multiarch-tuning-operator
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: x86_64
+      ADDITIONAL_WORKER_VM_TYPE: n2-standard-4
+      ADDITIONAL_WORKERS: "1"
+      COMPUTE_NODE_REPLICAS: "2"
+      COMPUTE_NODE_TYPE: t2a-standard-4
+      OCP_ARCH: arm64
+    test:
+    - ref: ipi-install-heterogeneous
+    - ref: multiarch-tuning-operator-e2e-olm-kustomize
+    workflow: ipi-gcp
+- always_run: false
+  as: e2e-aws-ovn-proxy-mto-origin
+  capabilities:
+  - arm64
+  cron: 30 15 * * 2
+  steps:
+    cluster_profile: aws
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-initial
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: x86_64
+      ADDITIONAL_WORKER_VM_TYPE: m6i.xlarge
+      ADDITIONAL_WORKERS: "1"
+      COMPUTE_NODE_REPLICAS: "2"
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      MTO_OPERATOR_INSTALL_METHOD: catalog
```

Comment:
```suggestion
      MTO_BRANCH: v1.0
      MTO_OPERATOR_INSTALL_METHOD: catalog
```

Reply:
aha, thanks, I miss it in the new added config file